drivetrain.py

```diff
@@ -76,6 +98,10 @@ def steering(self, x, y):
 
         return left, right
 
+    def get_batt_v(self):

```

Comment:
Using the `@property` decorator would enable us to call the method `batt_v`, and then use it like a regular property (i.e. `self.batt_v`), which would be neater, I think

Reply:
Although, based on conversation, maybe `current_voltage` would be a better name...